model/perf.go

```diff
@@ -73,6 +73,10 @@ func CreatePerformanceResult(info PerformanceResultInfo, source []ArtifactInfo)
 func (result *PerformanceResult) Setup(e sink.Environment) { result.env = e }
 func (result *PerformanceResult) IsNil() bool              { return !result.populated }
 func (result *PerformanceResult) Find() error {
+	// Need to maintain anything that gets lost after populate result
+	//
+	env := result.env
```

Comment:
why?

Reply:
It's a bug not to restore env, cause then you manipulate `result` and `env` is missing. So if I call `result.Find()`, populate result with some additional data (such as aux data), then call result.Save() it will fail because the `env` field is nil.